tests/integrationv2/test_renegotiate_apache.py

```diff
@@ -33,6 +41,8 @@ def create_get_request(route):
     "endpoint", [CHANGE_CIPHER_SUITE_ENDPOINT, MUTUAL_AUTH_ENDPOINT]
 )
 def test_apache_endpoints_fail_with_no_reneg(managed_process, protocol, endpoint):  # noqa: F811
+    skip_for_openssl3_fips()
```

Comment:
I noticed newer apache refusing to start... in draft until the nixpkgs upgrade becomes more pressing: https://github.com/aws/s2n-tls/pull/5039

Reply:
Oh nice! I think just skipping is fine until then tho. I wasn't going to mess around regenerating the certs just for that use case :)